.github/workflows/performance_metrics_cron.yml

```
@@ -85,12 +72,16 @@ jobs:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-region: ${{ env.AWS_REGION }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          role-duration-seconds: 3600
+          role-duration-seconds: 14400
```

I like to add inline unit conversion but it's easily inspectable without the comment too.
```suggestion
          role-duration-seconds: 14400 # 4 hours
```